scripts/mkcloud

```diff
@@ -950,6 +950,8 @@ function cloudupgrade()
 {
     if iscloudver 5plus ; then
         crowbarupgrade_5plus
+        onadmin crowbar_shutdown_services
+        onadmin crowbar_openstack_backup
```

Comment:
these lines should go to the `crowbarupgrade_5plus`  function, as they are part of the crowbarupgrade